targets/mediawiki/MediaWikiTarget.php

```diff
@@ -40,6 +40,11 @@ public function install(): void {
     $cache_dir = $this->getSourceRoot().'/mw-cache';
     mkdir($cache_dir);
 
+    $file = __DIR__.'/LocalSettings.php';
```

Comment:
Just like with drupal7 this should modify the file after it has been copied to the tmp directory where the test is going to be run from.  (And use a pattern)